packages/flutter/lib/src/material/slider.dart

```diff
@@ -424,11 +413,30 @@ class _RenderSlider extends RenderBox implements SemanticsActionHandler {
   TextDirection _textDirection;
   set textDirection(TextDirection value) {
     assert(value != null);
-    if (_textDirection == value)
+    if (value == _textDirection)
```

Comment:
Does the style guide say which way to do these comparisons?

Reply:
Nope. I can change this to the other way. We had both written this method independently, this was the only difference.

I go back on forth on which I find more intuitive which is why I haven't put anything in the style guide...

Reply:
Actually I'm leaving it as `value == _textDirection` because that's what all the other setters in this file do.